### DISCUSSION TITLE
API Calls are stuck on pending

### DISCUSSION THREAD

**Discussion by richardchihe** [deepgram/community#338] [General help] [answered]
### Which Deepgram product are you using?

Deepgram API

### Details

After 11pm API calls suddenly gets stuck on pending, but API calls from Playground works well.

<img width="779" alt="Screen Shot 2023-09-07 at 1 35 45 AM" src="https://github.com/deepgram/community/assets/29975814/6e1868da-b52d-4108-9e3d-4f50e7b99e77">

**Comment by jpvajda** [comment, marked as answer]
@richardchihe  👋  are you still seeing these calls in a Pending state in the Deepgram Console?  Recently we implemented a change to our Console Logs when API calls that get stuck in pending. 

Eventually if they don't process we'll classify them as `Lost`.  Just let me know if you are seeing this change in state with these stuck API calls.

**Example** 👇 

![Screenshot 2023-09-11 at 5 45 39 PM](https://github.com/deepgram/community/assets/1395158/a618e541-265a-41a5-8ca4-656444de7ffd)
